FreeRTOS/FreeRTOS-Plus-TCP
# Applied PR #107 to DHCP /multi

Description
-----------
Applied the same type of changes as in PR #107: when there is no new network buffer, wait for e.g. 250 ms, and try again.

Test Steps
-----------
This is described in #107 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
